packages/nimble-components/src/table/styles.ts

```diff
@@ -38,11 +39,14 @@ export const styles = css`
         position: absolute;
         top: 0px;
         width: 100%;
+        height: var(--ni-private-table-scroll-height);
     }
 
     .table-row-container {
         width: 100%;
         position: relative;
+        grid-template-columns: var(--ni-private-table-row-grid-columns);
```

Comment:
This is a dead style and is being set here only for the inherit to work. See in chrome devtools how this property has no effect.

![image](https://user-images.githubusercontent.com/1588923/224168125-acabb9ad-4eff-4962-9032-507de55cadb1.png)


It being here unused just so it can be inherited by the child is difficult to understand / maintain. We should just remove this instead and use the css custom property directly as discussed above.